client/state/breadcrumb/reducer.js

```diff
@@ -0,0 +1,34 @@
+import {
+	BREADCRUMB_RESET_LIST,
+	BREADCRUMB_UPDATE_LIST,
+	BREADCRUMB_APPEND_ITEM,
+} from 'calypso/state/action-types';
+import { combineReducers, withSchemaValidation } from 'calypso/state/utils';
+import { breadcrumbSchema } from './schema';
+
+// Stores the complete list of breadcrumbs in an array,
+export const items = withSchemaValidation( breadcrumbSchema, ( state = [], action ) => {
```

Comment:
`withSchemaValidation` is also needed for persisting the `state`. We can remove it.

Reply:
Right, removed at https://github.com/Automattic/wp-calypso/pull/60491/commits/1814a84396098f3d617714d02c85292340a9de56